docs/dev_cycle.intro.rst

```diff
@@ -0,0 +1,134 @@
+Introduction
+============
+
+.. contents:: :local:
+
+Background
+----------
+
+The purpose of this tutorial is to go one step beyond the common "Hello World" Greeter example,
+to the entire developement cycle of an Ethereum smart contract. We will try to unpack
+the confusing issues, clear up the mystery, and have some fun when you dive into blockchain and contracts developemnt.
+
+The tools we will use are Populus and Web3.py
+
+.. note::
+
+    Web3 in general is the client side API that let you intreacts with the blockchain. Populus is a 
+    development framework, that is built on top of Web3. If you are into javascript, you can use the
+    `Truffle javascript framework <http://truffleframework.com/>`_, and Web3.js which ships with the
+    ``geth`` client. If you prefer Python, then Populus and Web3.py are your friends.
+    
+
+We assume that your read 1-2 intros about Ethereum and the blockchain, and know Python.
```

Comment:
Maybe: *"This tutorial assumes you have some very basic understanding of the ethereum blockchain as well as with the Python programming language"*